input_topic: pointcloud2_clustering/clusters
out_topic: pointcloud2_cluster_tracking/clusters
marker_topic: visualization_marker
marker_frame_id: "/base_link"

method: 2 #1 if you want to delete overlap from clusters and 2 if you do not want to delete the overlap from clusters

size: 2
overlap: 0.2
marker_flag: true # (bool)marker_flag=true : markers for centroids activated
maxHungDist: 700 
trackTheUntracked: true
minMotionDist: 1000.0
max_dist: 0.31 # when two clusters are closer from max_dist (meters)